guidelines/sc/22/target-size-minimum.html

```diff
@@ -7,7 +7,7 @@ <h4>Target Size (Minimum)</h4>
     
     <p>The size of the <a>target</a> for <a>pointer inputs</a> is at least 24 by 24 <a>CSS pixels</a>, except where:</p>
     <ul>
-        <li><strong>Spacing:</strong> The target does not overlap any other target and has a <a>target offset</a> of at least 24 CSS pixels to every adjacent target;</li>
+        <li><strong>Spacing:</strong> Targets are spaced such that if 24 CSS pixel diameter circles are centered on each target, none of the circles intersect another circle or target.</li>
```

Comment:
I think this scans a bit better
```suggestion
        <li><strong>Spacing:</strong> Targets are spaced such that if a 24 CSS pixel diameter circle is centered on each target, none of the circles intersect another circle or target.</li>
```

Reply:
I also think that we can make a small addition to take care of an edge case.

```suggestion
        <li><strong>Spacing:</strong> Targets are spaced such that if a 24 CSS pixel diameter circle is centered on each undersized target, none of the circles intersect another circle or target.</li>
```

Reply:
@mbgower Which edge case should make it necessary to insert "undersized"? My examples are not.

Reply:
@JAWS-test in the case of a large target with small target inside (with some spacing)  in the centre, their circles will overlap. Assuming the large target passes the primary line of the SC, we don't need to worry that it's circle overlaps the smaller target's circle.

Reply:
@alastc My examples were such that both targets are not large enough (see above)

Reply:
@JAWS-test please provide visuals, as offered. Your '2 bars' examples sounds to me like it's actually 3 or 4 targets, not two, and all of them seem to pass.

Reply:
For the tiny circle inside a big thin donut example, I can see a way to word this to cover this edge case, but it would begin to get pretty cumbersome ('centered within' or something like that that makes it clear that the center point has to actually be part of the target). As far as I can tell, the only odd scenario left with this wording is the one you mentioned -- a small target centered inside another small target with sufficient spacing such that risk of hitting the wrong one isn't increased. But I personally have no issue telling the owner of such a designer that they should just make their targets big enough. It's not a common pattern; the new wording makes common patterns much easier to assess (with understandable outcomes).

Reply:
@mbgower I have added the two figures above. The question is why the circle must be centered? In the example with the "radio button" (inner circle, circular ring) the strange thing is that it forbids the variant with the best click area, but allows if the inner circle is moved closer to the circular ring (see figure 3), although this is harder to click

Figure 3:

![target3](https://user-images.githubusercontent.com/52748763/224476143-68ede620-b9e7-4de4-b3ce-4b0c65dff453.JPG)

Reply:
@JAWS-test 
>  The question is why the circle must be centered?

We've already gone into that exploration before. If you remove the centered notion, you immediately end up with odd allowed edge cases too, not to mention the whole thing is a _lot_ more fiddly to test. 
As stated in other threads, it's really useful to look at these things at scale. I encourage you to draw these things at scale and ask yourself whether any of those are good designs.

Reply:
@mbgower 

> ask yourself whether any of those are good designs

I don't think that's the question, at least not for me as a WCAG tester. As a WCAG tester, I wonder much more why example 1 fails and example 3 passes, even though 1 is better than 3.

Reply:
> wonder much more why example 1 fails and example 3 passes, even though 1 is better than 3.

I think a better way to look at this is to phrase that as "even though 3 is an even worse design than 1". They are both pretty bad.

It is possible to come up with hypothetical edge cases that demonstrate shortcomings of pretty much every requirement. Whether _anyone_ would be well-served by someone implementing such a design is a valid question, IMO. Do we create SCs that seek to guide real-world design to a better outcome, or do we focus on preventing a bunch of unlikely edge cases by creating a wordy SC that is not easy to understand or implement? My firm belief is we should not allow the perfect to be the enemy of the good.

BTW, the metrics are such that for the uncentered circle in the ring to pass, it has to be big enough that the circle is about 20 pixels in diameter. It's nothing to celebrate, but since it's the same spacing needed for an undersized circle target close to a full-sized target, it is at least consistent, from that perspective.

<img src="https://user-images.githubusercontent.com/14298245/227232964-31e91714-3bd6-4a13-8a94-e8f9634fc218.png" height="90" alt="a dot off-center inside a ring at roughly actual size">